playbooks/ansible-cloud/appliance-base/pre.yaml

```diff
@@ -52,3 +52,10 @@
         write_inventory_dest: /etc/ansible/hosts
         write_inventory_exclude_hostvars:
           - ansible_connection
+
```

Comment:
we should move this into ansible-test-cloud-integration-base bootstrap yaml file.  As these changes are specific to ansible-test.

That allows us to establish this job is for getting node online in zuul, and ansible-test-cloud-integration-base for things needed to make ansible-test work.

Reply:
ok, make sense.